src/remind.ts

```diff
@@ -18,7 +18,9 @@ async function main() {
     q: `repo:electron/electron ${q}`,
     sort: 'created',
   });
-  if (items.length) {
+
+  // silence during quiet period
+  if (items.length && new Date().getMonth() < 11) {
```

Comment:
maybe pull this check out into a common `isQuietPeriod()` function?

Reply:
[3f84def](https://github.com/electron/zoilist/pull/17/commits/3f84def92d404630ffdbcd74de106c582523b6ce)